rticonnextdds_connector/rticonnextdds_connector.py

```diff
@@ -1000,6 +1042,32 @@ def wait(self, timeout=None):
 		retcode = connector_binding.waitForAcknowledgments(self.native, timeout)
 		_check_retcode(retcode)
 
+	def wait_for_match(self, timeout = None):
```

Comment:
I've changed my mind about the name of this function. I think wait_for_subscriptions is better.